mozilla/fx-private-relay
# MPP-2331 Static country code

This PR fixes MPP-2331
 

# New feature description

The country selector from the “Verify your true phone number” step of the phone number mask onboarding has display issues. This PR removes the country selection component and replaces it with a static +1 as seen in the updated designs in Figma. 

# Screenshot (if applicable)

<img width="485" alt="image" src="https://user-images.githubusercontent.com/3924990/187778706-5662bab5-dc2a-4b9b-8efb-3a119bc1e88d.png">

# How to test

1. Be signed in into a Premium Firefox Relay account;
2. Have the Phone Number Masks subscription purchased;
3. Be on the “Verify your true phone number” step of the phone number mask onboarding;
4. Observe input field

# Checklist
